locales/pt-BR

```diff
@@ -0,0 +1,51 @@
+SirTrevor.Locales.pt-BR = {
```

Comment:
Great! The only two things that need changing is:
- filename should have .js
- you have to replace this first line with: `SirTrevor.Locales.pt_BR` (can't use the dash)